helm/charts/scheduled-volume-snapshotter/crds/scheduled-volume-snapshot-crd.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
I think you also need to remove the `version: v1beta1` in favor of `versions` (https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/)